src/Microsoft.Azure.SignalR.Protocols/ServiceMessage.cs

```diff
@@ -224,6 +224,49 @@ public ServiceErrorMessage(string errorMessage)
         }
     }
 
+    /// <summary>
+    /// A service warning message
+    /// </summary>
+    public class ServiceWarningMessage : ExtensibleServiceMessage
+    {
+        /// <summary>
+        /// Gets or sets the type of warning object.
+        /// <list type="bullet">
+        /// <item>connection</item>
```

Comment:
shall the user care about the case? the case here is all lower case while the case for WarningKind is capitalized.

Reply:
use enum now.